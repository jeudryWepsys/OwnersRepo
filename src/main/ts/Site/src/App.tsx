import './App.scss';
import * as React from 'react';
import AppRouter from './AppRouter';
import './index.scss';

const App = () => (
  <div>
    <AppRouter />
  </div>
);

export default App;
