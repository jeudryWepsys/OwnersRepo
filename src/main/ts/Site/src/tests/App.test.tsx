// eslint-disable-next-line import/no-extraneous-dependencies
import { shallow } from 'enzyme';
import * as React from 'react';
import App from '../App';

export { };

describe('<App/> tests', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
});
