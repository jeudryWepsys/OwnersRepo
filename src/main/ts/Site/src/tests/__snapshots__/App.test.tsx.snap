// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> tests should render correctly 1`] = `
<div>
  <AppRouter />
</div>
`;
